test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGet.CommandLine.Test.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
     <UseParallelXunit>true</UseParallelXunit>
     <Description>An end-to-end test suite for NuGet.CommandLine. Contains tests for every nuget.exe CLI command. Overlaps in tests with NuGet.CommandLine.FuncTest.</Description>
+    <TestProjectType Condition="'$(IsXPlat)' != 'true'">Functional</TestProjectType>
```

Comment:
How does this run on mono now?

Reply:
No tests are currently running under Mono for this pipeline.  This functional test only runs on Windows.  To make it work on Mac for Mono, I'll need to explore building the `NuGet.CommandLine` project on Mac where I think the only technical challenge is running `ilmerge.exe`.  If that worked, I could get Mono tests running again but considering how deprecated Mono is and how similar it is to .NET Framework, I don't foresee us really needing Mono tests anymore.